### DISCUSSION TITLE
Move asdf-hashicorp here

### DISCUSSION THREAD

**Discussion by DustinChaloupka** [asdf-community/.github#14] [General]
We would like to move https://github.com/Banno/asdf-hashicorp here, not sure what that all entails (request from https://github.com/Banno/asdf-hashicorp/issues/18).

**Comment by smorimoto** [comment]
Cool! I'm glad to hear that. Could you please invite me as the administrator of the repository first? (Required to transfer a repository.) Then I will change Terraform's code (https://github.com/asdf-community/infrastructure) to create a team and invite you as maintainers.

**Reply by smorimoto** [reply]
(I will invite @radditude as well.)

**Reply by smorimoto** [reply]
I just realized that collaborator doesn’t have a permission to transfer the repository. Can you transfer it yourself?

**Reply by DustinChaloupka** [reply]
It has been transfered. :)

**Reply by smorimoto** [reply]
Cool! Thanks. I will import the repository to the Terraform.
